### ISSUE TITLE
Ethos screen swiping (almost) does not work with LUA images

### ISSUE THREAD

**Issue by i3dm** [FrSkyRC/ETHOS-Feedback-Community#1502]
Video:
https://youtu.be/I4Ecw_rBvsA

notes: 
1. All images in the LUA are loaded during run time, **nothing is loaded during init.**
2. all images are BMP565 formattewd using Hobby4Life Ethos BMP formatting utility.

pressing "page" button to change pages moves to the next widget screen which starts blank and then loads in about 2s which is lets say slow but acceptable.
but swiping screens became totally useless.

**expected result:**
swiping is smooth, to a widget blank screen (in worst case), and then the widget takes 2s to load graphics.

**actual result:**
swiping is practically impossible. gets stuck in the middle or not responding at all to several tries.

Example lua attached.
[GAS.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/8148534/GAS.zip)

**Comment by i3dm** [comment]
> > All images in the LUA are loaded during run time
> 
> Seriously?

Yes.

**Comment by bsongis-frsky** [comment]
Please provide a script to put some evidence on this issue. Your script is hard to follow, badly indented. It has many local values which are shared between instances of your script, i.e. GAScycleCounter is very badly used. 

I hope nobody uses it in this state.

**Comment by i3dm** [comment]
1. fixed local variables (widget attached)
2. removed all other widgets from SD
3. running just 1 widget instace

result: exactly the same:
https://www.youtube.com/watch?v=qRbaL-8Ene8
[GAS.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/8177733/GAS.zip)

**Comment by bsongis-frsky** [comment]
Yes it seems your paint is far too long and freezes the whole task for far too long
I will start thinking about a way to kill such scripts to avoid users think it's because of Ethos that the radio freezes

**Comment by i3dm** [comment]
what do you mean paint is far too long? why is there a limit to the length of the paint function?
